Sources/TensorFlow/Initializers.swift

```diff
@@ -412,21 +462,6 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 }
 
-public extension Tensor where Scalar: TensorFlowFloatingPoint {
-    /// Creates a tensor by performing Glorot uniform initialization for the specified shape,
-    /// randomly sampling scalar values from a uniform distribution between `-limit` and `limit`,
-    /// generated by the default random number generator, where limit is
-    /// `sqrt(6 / (fanIn + fanOut))` and `fanIn`/`fanOut` represent the number of input and output
-    /// features multiplied by the receptive field if present.
-    ///
-    /// - Parameters:
-    ///   - shape: The dimensions of the tensor.
-    init(glorotUniform shape: TensorShape, seed: TensorFlowSeed = Context.local.randomSeed) {
-        let uniform = Tensor(randomUniform: shape, seed: seed)
-        self = Tensor.glorot(fromStandardUniform: uniform, shape: shape)
-    }
-}
-
 // TODO: Can become fileprivate after the 0.4 release.
 internal extension Tensor where Scalar: TensorFlowFloatingPoint {
     static func glorot(
```

Comment:
You can move this function in `BackwardsCompatible.swift` since it's only used there. You can place it in a similar position there, right above the extensions that are using it.